server/schemas/integration.yml

```diff
@@ -143,7 +143,7 @@ paths:
         - bearerAuth: []
       summary: Returns a list of items.
       tags:
-        - Project items
+        - Items project
```

Comment:
this can help with catigorizing the APIs like this: 
![image](https://user-images.githubusercontent.com/81808708/233127239-f2012dce-2236-4e77-b846-dbd96c03609f.png)